knudev-security/src/main/java/ua/knu/knudev/knudevsecurity/security/filters/JwtAuthenticationFilter.java

```diff
@@ -46,7 +48,7 @@ protected void doFilterInternal(@NotNull HttpServletRequest request,
         if (accountUsername != null && SecurityContextHolder.getContext().getAuthentication() == null) {
             if (!jwtService.isAccessToken(jwt)) {
                 String message = "Please enter an access token";
```

Comment:
the same with message here, as i told above

Reply:
done